Source/WebCore/accessibility/ios/AccessibilityObjectIOS.mm

```diff
@@ -171,6 +176,37 @@
 #endif // HAVE (INLINE_PREDICTIONS)
 }
 
+static RetainPtr<NSDictionary> unarchivedTokenForData(RetainPtr<NSData> tokenData)
+{
+    NSError *error = nil;
+    return [NSKeyedUnarchiver unarchivedObjectOfClasses:[NSSet setWithObjects:[NSDictionary class], [NSNumber class], [NSString class], nil] fromData:tokenData.get() error:&error];
+}
+
+std::span<const uint8_t> AXRemoteFrame::generateRemoteToken() const
+{
+    return span(Accessibility::newAccessibilityRemoteToken([[NSUUID UUID] UUIDString]));
+}
+
+void AXRemoteFrame::initializePlatformElementWithRemoteToken(std::span<const uint8_t> token, int processIdentifier)
+{
+    m_processIdentifier = processIdentifier;
+
+    RetainPtr nsToken = toNSData(token);
+    NSDictionary *tokenDictionary = nsToken ? unarchivedTokenForData(nsToken).get() : nullptr;
+    if (!tokenDictionary)
+        return;
+
+    NSString *uuid = [tokenDictionary objectForKey:@"ax-uuid"];
+    AXRemoteElement* remoteElement = [allocAXRemoteElementInstance() initWithUUID:uuid andRemotePid:processIdentifier andContextId:0];
+    remoteElement.onClientSide = YES;
+    remoteElement.accessibilityContainer = parentObjectUnignored() ?  parentObjectUnignored()->wrapper() : nil;
```

Comment:
It would be a tiny bit more efficient to only get `parentObjectUnignored` only once, i.e.:

```cpp
RefPtr parent = parentObjectUnignored();
remoteElement.accessibilityContainer = parent ? parent->wrapper() : nil;
```